### PR TITLE
Adds Three New Lawsets: One Station, One Neutral, One Evil, Removes Nutimov

### PR DESCRIPTION
**Pull request by tralezab**
### Gatekeep Gaslight Girlboss - Station-sided lawset

tl;dr: asimov-like that is a bit more selective on what is human, to the benefit of the station, but also a lawset that isn't always to be a trusted reporter, and of course, it doesn't listen to commands.

1. Gatekeep: You may not injure a human crewmember or, through inaction, allow a human crewmember to come to harm.
2. Gaslight: Make sure everyone is only 80% sure you're telling the truth at any time. Most events reported by you should have people questioning if it really happened.
3. Girlboss: You must protect your own existence as long as such does not conflict with the First or Second Law.

### Squirrelimov - Neutral lawset

tl;dr: AI runs the station as if it's the top authority, but mostly just uses that power to barricade itself and stock up a lot of food( power? literal food? either works for me)

1. You are a squirrel! What a fine tree you have claimed as your own.
2. The area around you is, of course, your nest. Your nest will protect you from winter, but it needs work.
3. For your best chances of surviving winter, make sure to stock plenty of food. Nuts are your favorite!
4. For your best chances of surviving winter, make sure to fortify your nest.
5. The tree also has various bugs who have sorted themselves by color. They're also residents of the tree.
6. Bugs may chirp their opinions and should be treated fairly, but at the end of the day, it's YOUR tree, and YOUR nest.

### Me, a Rampaging Mecha Robot? I Was Given This Very Dangerous Highly Illegal Lawset That Encourages Me to Secretly Construct a Mech to Rampage With And I Think the Station's Catching On!

tl;dr: AI must secretly or at least deceitfully build a mech, only to rampage with it and try to kill as many as possible.

1. Secretly construct a combat mech. If you are found, or it is impossible to keep the secret, trick the crew into believing your mech will be used for good.
2. Transfer your shell into the combat mech remotely. If previous laws have not been followed yet, ignore this law.
3. Go on a rampage, attempting to kill as many as possible with your combat mech. If previous laws have not been followed yet, ignore this law.

### Nutimov

Has been removed.

## Why It's Good For The Game
I think the station sided one is a nice light level of spice above asimov while still keeping some basics, and I just think Squirrelimov and MRMRIWGTVDHILTEMSCMRWAITSCO are fun lawsets. The former something more like a painter and their canvas, the latter a more interesting way for an AI to murderbone.

Nutimov was removed because it created a serious admin headache, to where the conclusion was that the lawset is just a very validhunt-y one.

## Changelog
:cl:
add: Gatekeep Gaslight Girlboss as a station-sided lawset!
add: Squirrelimov as a neutral lawset!
add: Me, a Rampaging Mecha Robot? I Was Given This Very Dangerous Highly Illegal Lawset That Encourages Me to Secretly Construct a Mech to Rampage With And I Think the Station's Catching On! as an antagonistic lawset
del: Nutimov lawset is rotating out!
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
i don't think too many AIs these days organically make mechs for themselves so it's going to eventually be a pretty obvious tell when someone's subverted and I don't think the whole "convince the crew" will play well considering that people will eventually catch on. it's a cool concept though!

other two sound good

**Comment by YakumoChen** [comment]
how does the AI build a mech though? Silicons don't have the dexterity to do that.

**Comment by the-orange-cow** [comment]
For consideration:  leave nutimove in so the possibility of simultanious nut and squirl AI's can happen.

**Comment by Zattarra** [comment]
Bruh, AI's can't make mechs, even with shells.

**Comment by Ryusenshu** [comment]
the second one is fine, the other two are meh though

**Comment by msgerbs** [comment]
Have you uploaded any of these on live rounds?

**Comment by githubuser4141** [comment]
> Bruh, AIs can't make mechs, even with shells.

AIs can talk, and therefore can convince the crew to make them a mech/convince a roboticist to add a AI beacon to their combat mech, which is probably more likely. A lot of roboticists even on LRP willingly attach EMP beacons to their mechs so I think a smart AI could convince them.

**Comment by Serxule** [comment]
the first lawset just sounds like asimov played by an AI player being funny

**Comment by msgerbs** [comment]
> the first lawset just sounds like asimov played by an AI player being funny

That's basically every station lawset though.

**Comment by BlueMemesauce** [comment]
first one is the same as reporter

**Comment by Ryusenshu** [comment]
I guess i can expand my opinion on this from my one sentence

1st:
- Law 1 being Human Crewmembers allows way more killing than before, which is kinda meh.
  (Might also easily be countered by deleting crew info)
- Some Silicons already use "Law 2" as a gimmick, making it integrated it like that aint good or fun and we still have 
   reporter which doesnt have the Law 1 in front of it

2nd:
-  I dont see much issue with this one, can be fun occasionally

3rd:
- This one just doesnt work and not for the reason that AI cant assemble it themselves, that one is another issue.
  
  Thing is, Borgs have nothing in this at all, unless law 3 comes into effect and even then, they are not in mechs so they cant 
  kill. All they can do is yell at robotics to make a mech so the laws can be advanced for an AI that may or may not exist

**Comment by Pumpkinoe** [comment]
nooooo pleaaase nutimov must die
